src/en/Octopii.lua

```diff
@@ -98,8 +96,38 @@ return {
 	end,
 
 	hasSearch = true,
-	isSearchIncrementing = true,
+	isSearchIncrementing = false,
 	search = function(data)
-		return parseTop(GETDocument(expandURL("/page/" .. data[PAGE] .. "/?s=" .. encode(data[QUERY]))))
+        -- Create JSON body for POST request.
+        local requestJson = "{" ..
+                                '"clicked":false,' ..
+                                '"limit":"200",' .. -- increased limit to limit to a single request
+                                '"page":0,' .. -- therefore, only request first page
+                                '"pageCount":1,' ..
+                                '"value":"' .. data[QUERY] .. '",' ..
+                                '"sort":4,' ..
+                                '"selected":{"genre":[],"status":[],"sort":[],"author":[]},' ..
+                                '"results":[],' ..
+                                '"label":"searching ...."' ..
+                            "}"
+        local body = RequestBody(requestJson, MTYPE)
+        local post = RequestDocument(POST("https://octopii.co/api/advance-search", nil, body)):toString()
+
+		print(post)
+        -- Cut HTML body from the actual JSON answer and decode it.
+        post = string.sub(post, 33, string.len(post)-17)
```

Comment:
```suggestion
        post = post:sub(33, -18)
```
Or consider using jsoup or `:match` to avoid issues if they change the surrounding HTML.